pages/community/precice-workshop-organization.md

```diff
@@ -0,0 +1,116 @@
+
+In this page, we document common steps and lessons learned, aiming to make onboarding of new organizers easier. Besides, who remembers what material the registration desk needed last year, if we cannot even remember what we had for lunch?
+
+## First urgent steps
+
+Do these steps as soon as possible, as other major steps depend on them.
+
+### Set and advertise a date
+
+Decide on a place and date before the previous year's workshop, so that this can be announced to the current participants.
+
+Factors that typically contribute to a date decision:
+
+- Venue availability
+- Lecture-free periods
+- School holidays
+- Overlap with other related conferences (check, e.g., ECCOMAS, IACM, SIAM)
+
+Once finalized, a "save the date" entry should be listed immediately on:
+
+- the preCICE website (news banner, community overview, first dummy page of the workshop)
+- the preCICE forum
+- the social media platforms that preCICE is active on
+
+### Find a venue
+
+If not at the same time as the date announcement, decide on a venue, and book rooms as soon as possible after that. Depending on the local workflows and availability, room booking requests might need to be done more than six months ahead, so aim to book at least the main room at the time that the venue is decided.
+
+Typical room requirements:
+
+- Fitting around 50 people (but not much larger than that)
+- For the hands-on training course and the user support sessions: easy access at every seat, (collaborative) working surface, enough power sockets, stable internet connection
+- For the talks: good acoustics and visibility, recording infrastructure
+- For the world café: (standing) tables, placed far from each other, isolated from noisy areas of the building
+- For the catering: ideally next to the rooms, isolated, standing
+
+With "recording infrastructure", we mean infrastructure to capture the speaker audio+video, slides, and an additional microphone in separate stream (a meeting camera system is typically not fitting the purpose). We can provide specialized hardware to do the mixing of these channels into what you later see as recordings on YouTube, if this is not already available in the room.
+
+The rooms for the training and the talks are typically different, even if the schedule is typically structured as a single track. The catering and the world café can typically use the same room and tables, but reusing the collaborative working room for the world café is also fine.
+
+### Define financing scheme
+
+Typically, the workshop is funded by participation fees. An important dependency for subsequent steps is which entity can collect and process such participation fees. This might be an association supporting scientific activities, an event management unit of a university, or the organizing institute (e.g., via project infrastructure).
```

Comment:
```suggestion
Typically, the workshop is co-funded by participation fees. An important dependency for subsequent steps is which entity can collect and process such participation fees. This might be an association supporting scientific activities, an event management unit of a university, or an external company. The setup should allow to model an income to reinvest into maintaining preCICE.
```
How is "event management unit of a university" different from "organizing institute"?

Reply:
I meant that there might be a dedicated service at the university that supports organizing events, instead of the chair itself organizing this by themselves without institutional support.

But this is a better formulation.